lib/discordrb/events/members.rb

```diff
@@ -83,6 +83,12 @@ class ServerMemberDeleteEvent < ServerMemberEvent
     def init_user(data, bot)
       @user = Discordrb::User.new(data['user'], bot)
     end
+
+    # @!attribute [r] user
+    #   @return [User] the user in question.
+    def user
+      super
+    end
```

Comment:
```suggestion
   # @return [User] the user in question.
   attr_reader :user
```
This is the format z64 requested